### PR TITLE
reference/tools: add solution for argument-list too long error for DM 1.0.2

### PR DESCRIPTION
**Pull request by lichunzhu**
### What is changed, added or deleted? 
From DM 1.0.2 on we will generate table list automatically. When generated table-list is too long will cause DM's error. The error handle docs should be updated.

### What is the related PR or file link(s)? 
https://github.com/pingcap/dm/pull/310

### Which version does your change affect? 
update dev now, and after get two LGTM I will update v2.1 and v3.0

### CODE REVIEW

File: dev/reference/tools/data-migration/error-system.md [hunk 1]
```diff
@@ -124,4 +124,5 @@ DM 根据错误的严重程度和必要性来选择是否输出错误堆栈。
 | `code=11006` |  DM 内置的 parser 解析不兼容的 DDL 时出错              |  可参考 [Data Migration 故障诊断-处理不兼容的 DDL 语句](/dev/how-to/troubleshoot/data-migration.md#处理不兼容的-ddl-语句) 提供的解决方案 |
 | `code=20010` | 处理任务配置时，解密数据库的密码出错                             |  检查任务配置中提供的下游数据库密码是否有[使用 dmctl 正确加密](/dev/how-to/deploy/data-migration-with-ansible.md#使用-dmctl-加密上游-mysql-用户密码) |
 | `code=26002` | 任务检查创建数据库连接失败。更详细的错误信息可通过错误 message 获取。错误 message 中包含操作数据库所返回的错误码和错误信息。 |  检查 DM-master 所在的机器是否有权限访问上游 |
+| `code=32001` | Dump 部件异常                                              | 如果报错 `msg` 包含 `mydumper: argument list too long.`，则需要用户根据 black-white list 手动在 task.yaml 的 mydumper extra-args 参数中加上 --regex 正则表达式设置要导出的库表：例如要导出所有库中表名字为 hello 的表可加上 `--regex '.*\\.hello$'`，要导出所有表可加上 `--regex '.*'`。 |
```
Comment (csuzhangxc) [inline dev/reference/tools/data-migration/error-system.md]:
```suggestion
| `code=32001` | dump 处理单元异常                                            | 如果报错 `msg` 包含 `mydumper: argument list too long.`，则需要用户根据 black-white list 手动在 `task.yaml` 的 mydumper `extra-args` 参数中加上 `--regex` 正则表达式设置要导出的库表：例如要导出所有库中表名字为 `hello` 的表可加上 `--regex '.*\\.hello$'`，要导出所有表可加上 `--regex '.*'`。 |
```

**Comment by csuzhangxc** [comment]
@lilin90 PTAL

**Comment by lichunzhu** [comment]
@lilin90 PTAL

File: dev/reference/tools/data-migration/troubleshoot/error-handling.md [hunk 2]
```diff
@@ -19,6 +19,7 @@ category: reference
 | `code=11006` |  DM 内置的 parser 解析不兼容的 DDL 时出错              |  可参考 [Data Migration 故障诊断-处理不兼容的 DDL 语句](/dev/reference/tools/data-migration/faq.md#处理不兼容的-ddl-语句) 提供的解决方案 |
 | `code=20010` | 处理任务配置时，解密数据库的密码出错                             |  检查任务配置中提供的下游数据库密码是否有[使用 dmctl 正确加密](/dev/how-to/deploy/data-migration-with-ansible.md#使用-dmctl-加密上游-mysql-用户密码) |
 | `code=26002` | 任务检查创建数据库连接失败。更详细的错误信息可通过错误 message 获取。错误 message 中包含操作数据库所返回的错误码和错误信息。 |  检查 DM-master 所在的机器是否有权限访问上游 |
+| `code=32001` | dump 处理单元异常                                            | 如果报错 `msg` 包含 `mydumper: argument list too long.`，则需要用户根据 black-white list 手动在 `task.yaml` 的 mydumper `extra-args` 参数中加上 `--regex` 正则表达式设置要导出的库表：例如要导出所有库中表名字为 `hello` 的表可加上 `--regex '.*\\.hello$'`，要导出所有表可加上 `--regex '.*'`。 |
```
Comment (lilin90) [inline dev/reference/tools/data-migration/troubleshoot/error-handling.md]:
```suggestion
| `code=32001` | dump 处理单元异常                                            | 如果报错 `msg` 包含 `mydumper: argument list too long.`，则需要用户根据 black-white list，在 `task.yaml` 的 mydumper `extra-args` 参数中手动加上 `--regex` 正则表达式设置要导出的库表。例如，如果要导出所有库中表名字为 `hello` 的表，可加上 `--regex '.*\\.hello$'`，如果要导出所有表，可加上 `--regex '.*'`。 |
```

**Comment by lichunzhu** [comment]
Updated, PTAL @lilin90

**Pull request merged**